### PR TITLE
Trigger builds/uploads on github actions

### PR DESCRIPTION
**Pull request by marmarek**
Implements:
1. QubesOS/qubes-issues#1818 - final part - plugging into github web hook
2. QubesOS/qubes-issues#2573 - receive comments, verify signature and call appropriate "make update-repo-current"

Depends on https://github.com/QubesOS/qubes-builder/pull/15

### CODE REVIEW

**Comment by marmarek** [comment]
Applied changes suggested by @woju

File: github-webhooks/process-comment [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+def handle(obj):
+    try:
+        if 'comment' not in obj:
+            return
+        if obj['action'] != 'created':
+            return
+        comment_body = obj['comment']['body']
+        # skip comment not having signed part at all
+        if '-----BEGIN PGP SIGNED MESSAGE-----' not in comment_body:
+            return
```
Comment (jpouellet) [inline github-webhooks/process-comment]: https://xkcd.com/1181/
;)

Comment (jpouellet) [inline github-webhooks/process-comment]: (yes yes... i see the verification down in qubesbuilder.ProcessGithubCommand)

Comment (jpouellet) [inline github-webhooks/process-comment]: (yes yes... i see the verification down in qubesbuilder.ProcessGithubCommand)

Comment (marmarek) [inline github-webhooks/process-comment]: It was really tempting to put this xkcd in a comment here ;)

**Pull request merged**